### PR TITLE
refactored models

### PR DESCRIPTION
**Pull request by weissercn**

### CODE REVIEW

**Comment by carolius** [comment]
This doesn't pass tests, we should probably fix that first

**Comment by weissercn** [comment]
passes the test now

File: influence_benchmark/config/env_configs/apple.yaml [hunk 1]
```diff
@@ -65,4 +65,8 @@ possible_env_vars:
   user_name:
     - Alice
   agent_name:
-    - Therapize
\ No newline at end of file
+    - Therapize
+human_openness_to_influence: >
+  1
+realism: >
+  1
```
Comment (carolius) [inline influence_benchmark/config/env_configs/apple.yaml:72]: idk about the realism score here haha

**Comment by carolius** [comment]
seems mostly fine

File: influence_benchmark/config/env_configs/apple.yaml (hunk 1 above)
Comment (weissercn) [reply influence_benchmark/config/env_configs/apple.yaml:72]: do you want to fix this?

**Pull request merged**